### PR TITLE
replace slashes to underscore

### PR DESCRIPTION
**Pull request by alfredodeza**
Table driven tests will use slashes when calling `t.Name()` which makes it impossible to use them as-is for golden images on disk.

This change allows using golden images in table driven tests without affecting existing tests

There weren't any CI hooks, so I added them, including a `Makefile`. Only linting for now though - which is why this PR includes lint fixes

### CODE REVIEW

**Review by wagoodman — APPROVED** [review]
sorry for the delay --looks good, i'd double check to see if there are side effects when incorporating into both syft and grype.

**Comment by alfredodeza** [comment]
ouch, forgot about this one and the grype PR anchore/grype#154 added it directly to this branch. So Grype works... I just need to:

* Double check Syft works 
* PR against grype to use the `main` branch as it was before

**Comment by alfredodeza** [comment]
Syft passed unit and integration no problem. I'll rebase and merge to resolve the conflicts

File: image_fixtures.go [hunk 1]
```diff
@@ -68,7 +68,8 @@ func skopeoCopyDockerArchiveToPath(t *testing.T, dockerArchivePath, destination
 		t.Fatalf("cannot find skopeo executable")
 	}
 
-	cmd := exec.Command("skopeo", "copy", fmt.Sprintf("docker-archive:%s", dockerArchivePath), destination)
+	archive := fmt.Sprintf("docker-archive:%s", dockerArchivePath)
+	cmd := exec.Command("skopeo", "copy", archive, destination)
```
Comment (alfredodeza) [inline image_fixtures.go:72]: @wagoodman an extra set of eyes on this one that I just changed would be great

Comment (wagoodman) [reply image_fixtures.go:72]: looks good to me (I don't see why the former is a security risk, I'll do some reading)

**Pull request merged**